config/amazon_config.template_json

```diff
@@ -1,4 +1,5 @@
 {
+  "instance_name": "Fairgame1",
```

Comment:
Why not have the default be Fairgame? Would the user have to edit this config file for each instance? What happens if you run two instances with the same name?

Reply:
Default instance name changed to Fairgame. 
User does not have to edit the default if they don't want to. But if running multiple instances then they can update it to help keep control of instances. 
Nothing will break if you run two instances with the same name.